package.json

```diff
@@ -47,13 +47,13 @@
     "lodash.escaperegexp": "^4.1.2",
     "lodash.mapvalues": "^4.6.0",
     "minimatch": "^3.0.4",
-    "node-cache": "^4.2.1",
+    "node-cache": "^5.1.0",
     "path-to-regexp": "^1.8.0",
     "primus": "^6.0.1",
     "primus-emitter": "^3.1.1",
     "qs": "^6.9.1",
     "request": "^2.88.0",
-    "snyk-config": "^2.2.3",
+    "snyk-config": "^3.1.1",
```

Comment:
no need in package-lock.json update?

Reply:
There's no `package-lock.json` because this is a library.

It doesn't have `package-lock=false` in `.npmrc`, which it probably should have.